### DISCUSSION TITLE
Lab chat setting context to cli_helper ansers all questions using python prompt

### DISCUSSION THREAD

**Discussion by konan-abhi** [instructlab/instructlab#818] [Model Behavior]
This happens intermittently though:

>>> /c cli_helper                                                                                                                     [S][default]
╭──────────────────────────────────────────────────────────────────── system ────────────────────────────────────────────────────────────────────╮
│ Welcome to Chat CLI w/ GGML-MERLINITE-7B-0302-Q4_K_M (new session)                                                                             │
╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
>>> print 12 months of a year                                                                                                      [S][cli_helper]
╭──────────────────────────────────────────────────────── ggml-merlinite-7b-0302-Q4_K_M ─────────────────────────────────────────────────────────╮
│ ```python                                                                                                                                      │
│ months = ['January', 'February', 'March', 'April', 'May', 'June',                                                                              │
│           'July', 'August', 'September', 'October', 'November', 'December']                                                                    │
│ ```                                                                                                                                            │
╰─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────── elapsed 15.763 seconds ─╯
>>> list 12 months of a year                                                                                                       [S][cli_helper]
╭──────────────────────────────────────────────────────── ggml-merlinite-7b-0302-Q4_K_M ─────────────────────────────────────────────────────────╮
│ ```python                                                                                                                                      │
│ import calendar                                                                                                                                │
│ calendar.month_name[1:13]                                                                                                                      │
│ ```                                                                                                                                            │
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────── elapsed 5.195 seconds ─╯
>>> what is 2+2                                                                                                                    [S][cli_helper]
╭──────────────────────────────────────────────────────── ggml-merlinite-7b-0302-Q4_K_M ─────────────────────────────────────────────────────────╮
│ ```python                                                                                                                                      │
│ 2 + 2                                                                                                                                          │
│ ```                                                                                                                                            │
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────── elapsed 2.634 seconds ─╯
>>> which century is this                                                                                                          [S][cli_helper]
╭──────────────────────────────────────────────────────── ggml-merlinite-7b-0302-Q4_K_M ─────────────────────────────────────────────────────────╮
│ ```python                                                                                                                                      │
│ current_year = int(datetime.now().year)                                                                                                        │
│ century = int(current_year / 100) if current_year > 0 else 19                                                                                  │
│ ```                                                                                                                                            │
╰─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────── elapsed 10.974 seconds ─╯
>>> what is a ball                                                                                                                 [S][cli_helper]
╭──────────────────────────────────────────────────────── ggml-merlinite-7b-0302-Q4_K_M ─────────────────────────────────────────────────────────╮
│ ```python                                                                                                                                      │
│ def what_is_a_ball():                                                                                                                          │
│     pass                                                                                                                                       │
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────── elapsed 4.215 seconds ─╯
>>>

**Comment by xukai92** [comment]
the current Merlinite-7B has no skill to do `cli_helper` actually. could be a good skill to contribute.
